### DISCUSSION TITLE
Panther not running my forward runs

### DISCUSSION THREAD

**Discussion by d-strom** [gmdsi/GMDSI_notebooks#112] [Q&A]
Hi,

When im starting a pest run with the command:

`pyemu.os_utils.start_workers(tmp_d, # the folder which contains the "template" PEST dataset
                            'pestpp-glm', #the PEST software version we want to run
                            'mod0000.pst', # the control file to use with PEST
                            num_workers=num_workers, #how many agents to deploy
                            worker_root='.', #where to deploy the agent directories; relative to where python is running
                            master_dir=m_d, #the manager directory`

I just get an anaconda window with the following appearence:
![image](https://github.com/gmdsi/GMDSI_notebooks/assets/74509984/76a90255-06d6-4147-bb35-d8281ea6810b)

That is, it seems frozen and do not start the workers. I have successfully run similar scripts in the past but now it just doesnt seem to work.

Does anyone have the slightest idea of what is going on and how to fix it?

Sincerely
D

**Comment by d-strom** [comment]
complementary info from the panther_worker .rec-file

![image](https://github.com/gmdsi/GMDSI_notebooks/assets/74509984/ca3a6546-fdb0-4218-9a1c-3128fb60ea54)

**Comment by jtwhite79** [comment]
Is the clock on the master output window still counting forward?

**Reply by d-strom** [reply]
Yeah, its counting. But it seems to be loosing connection to the master.

**Comment by jtwhite79** [comment]
Hmm.  If you reboot and retry does the same issue happen again (sorry, IT response!)

**Reply by d-strom** [reply]
Seems like there is something wrong with the set-up... for trouble shooting i revisited and ran one of the tutorial notebooks, which worked fine.

I havnt yet figured out what the cause is but will keep looking into it.

Thanks so far.

**Comment by d-strom** [comment]
Update: It was just a clumsy user-error. I had used a non-allowed sign in the file path... Atleast sometimes the solution is easy.
